src/assets/i18n/en.json

```diff
@@ -77,6 +77,8 @@
   "login.form.password.validation": "Password missing",
   "login.forgot_password": "Forgotten your password?",
   "login.forgot_password.error": "Unfortunately, your password could not be updated. Click here to try again",
+  "login.forgot_password.confirmation": "A password reset message has been sent to you!",
```

Comment:
Since the API always returns `HTTP 202 Accepted` we should probably write something a long the lines of "If an account exists for this email a password reset..."